src/elements/content-sidebar/activity-feed/common/delete-confirmation/DeleteConfirmation.js

```diff
@@ -50,15 +51,26 @@ class DeleteConfirmation extends React.Component<Props> {
                 role="dialog"
                 shouldDefaultFocus
                 shouldOutlineFocus={false}
+                {...rest}
```

Comment:
Since this is shared across tasks and comments, `data-resin-component` can be added here to distinguish events.